volestipy/tests/test_svd_rounding.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3.6
```

Comment:
please add library headers in new files

Reply:
This is the way you can run a python scripty just by making it executable and then run ```./<script>.py```
It is useful to me when I run the tests in terms of developing though
It would be better to remove them

Reply:
ok please remove